Controller/RegistrationFOSUser1Controller.php

```diff
@@ -64,13 +65,14 @@ public function registerAction()
                     $this->get('session')->remove('sonata_basket_delivery_redirect');
                     $url = $this->generateUrl($route);
                 } else {
-                    $url = $this->get('session')->get(
-                        'sonata_user_redirect_url',
-                        $this->generateUrl('sonata_user_profile_show')
```

Comment:
Not sure if I understand this change, AFAIK this second parameter purpose is to be used as a fallback when `get('sonata_user_redirect_url')` returns an empty value

http://api.symfony.com/2.8/Symfony/Component/HttpFoundation/Session/Session.html#method_get